internal/ircserver/cmd_privmsg.go

```diff
@@ -59,6 +59,21 @@ func (i *IRCServer) cmdPrivmsg(s *Session, reply *Replyctx, msg *irc.Message) {
 			Params:  []string{msg.Params[0], msg.Trailing()},
 		})
 		return
+	} else if strings.HasPrefix(msg.Params[0], "$") {
+		if msg.Params[0] == "$"+i.ServerPrefix.Name || msg.Params[0] == "$*" {
+			i.sendAllUsers(reply, &irc.Message{
```

Comment:
Please add a check here that the user who sent the command is actually an IRC operator — otherwise, every connected user could send messages to all other users :)

Example: https://github.com/robustirc/robustirc/blob/b1a793fbb7f321f06999a8b846eee288a9e81699/internal/ircserver/cmd_gline.go#L17-L25

Reply:
Woops, yes I missed that, thanks for the snippet!